<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="optionList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="optionPages">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="syntaxHighlighting">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="labelStyle">
             <property name="text">
              <string>Style :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="styleList">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>50</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="labelFont">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>&amp;Font :</string>
                 </property>
                 <property name="buddy">
                  <cstring>fontFamilly</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFontComboBox" name="fontFamilly"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="leftMargin">
                <number>40</number>
               </property>
               <item>
                <widget class="QLabel" name="labelFontSize">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>&amp;Size :</string>
                 </property>
                 <property name="buddy">
                  <cstring>fontSize</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="fontSize"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QCheckBox" name="fontBold">
               <property name="text">
                <string>&amp;Bold</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="fontItalic">
               <property name="text">
                <string>&amp;Italic</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="fontUnderline">
               <property name="text">
                <string>&amp;Underline</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLabel" name="syntaxExemple">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>80</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);
border-width: 1px;
border-style: solid;
border-color: black;</string>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QGridLayout" name="gridLayout">
               <property name="leftMargin">
                <number>40</number>
               </property>
               <property name="rightMargin">
                <number>40</number>
               </property>
               <item row="1" column="1">
                <widget class="ColorPicker" name="backgroundColorButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>B&amp;ackgroud color</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="margin">
                  <number>5</number>
                 </property>
                 <property name="buddy">
                  <cstring>backgroundColorButton</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>F&amp;oregroud color</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="margin">
                  <number>5</number>
                 </property>
                 <property name="buddy">
                  <cstring>foregroundColorButton</cstring>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="ColorPicker" name="foregroundColorButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="labelExemple">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Exemple :</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="resetStyle">
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="resetAllStyle">
               <property name="text">
                <string>Reset all</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>60</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QPushButton</extends>
   <header>colorpicker.h</header>
   <slots>
    <signal>colorSelected(QColor)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>optionList</tabstop>
  <tabstop>styleList</tabstop>
  <tabstop>fontFamilly</tabstop>
  <tabstop>fontSize</tabstop>
  <tabstop>fontBold</tabstop>
  <tabstop>fontItalic</tabstop>
  <tabstop>fontUnderline</tabstop>
  <tabstop>foregroundColorButton</tabstop>
  <tabstop>backgroundColorButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optionList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>changePage(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontSize</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>changeFontSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>835</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>847</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontFamilly</sender>
   <signal>currentFontChanged(QFont)</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>changeFont(QFont)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontBold</sender>
   <signal>toggled(bool)</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>toggleFontBold(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontItalic</sender>
   <signal>toggled(bool)</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>toggleFontItalic(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontUnderline</sender>
   <signal>toggled(bool)</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>toggleFontUnderline(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>443</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>styleList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>changeStyle(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>foregroundColorButton</sender>
   <signal>colorSelected(QColor)</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>changeFontForegroudColor(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>847</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backgroundColorButton</sender>
   <signal>colorSelected(QColor)</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>changeFontBackgroudColor(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>850</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetStyle</sender>
   <signal>clicked()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>resetStyle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>719</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>848</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetAllStyle</sender>
   <signal>clicked()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>resetAllStyle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>794</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>844</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changePage(int)</slot>
  <slot>changeFontSize()</slot>
  <slot>changeFont(QFont)</slot>
  <slot>toggleFontBold(bool)</slot>
  <slot>toggleFontItalic(bool)</slot>
  <slot>toggleFontUnderline(bool)</slot>
  <slot>changeFontForegroudColor(QColor)</slot>
  <slot>changeFontBackgroudColor(QColor)</slot>
  <slot>changeStyle(int)</slot>
  <slot>resetStyle()</slot>
  <slot>resetAllStyle()</slot>
 </slots>
</ui>
